packages/element-lib/package.json

```diff
@@ -72,6 +72,7 @@
   },
   "devDependencies": {
     "@transmute/eslint-config": "0.1.0",
+    "did-method-key": "^0.3.0",
```

Comment:
beware that this upgrade changed the way that publicKeyIds are represented to `did:example:123#kid` from just reusing the did as the publicKeyId.

Reply:
Looks like this isn't used, forgot to remove it